diff --git a/dist/util/check-system.js b/dist/util/check-system.js
index 9ed210be019a0028a6c5e0f4941c7d8930d35e6e..db741a903a3e16d738f3c334f3db68f095a79c6b 100644
--- a/dist/util/check-system.js
+++ b/dist/util/check-system.js
@@ -31,7 +31,7 @@ const NPM_ALLOWLISTED_VERSIONS = {
     linux: '>= 5.4.0',
 };
 const YARN_ALLOWLISTED_VERSIONS = {
-    all: '^1.0.0',
+    all: '>=1.0.0',
 };
 function checkValidPackageManagerVersion(packageManager, version, allowlistedVersions) {
     if (!semver_1.default.valid(version)) {
